src/include/duckdb.h

```diff
@@ -3083,6 +3083,17 @@ References the `from` vector in the `to` vector, this makes take shared ownershi
 */
 DUCKDB_C_API void duckdb_vector_reference_vector(duckdb_vector to_vector, duckdb_vector from_vector);
 
+/*!
+Returns the debug string from the data chunk.
+The string returned must be freed.
+*/
+DUCKDB_C_API const char *duckdb_data_chunk_to_string(duckdb_data_chunk chunk);
```

Comment:
Should this be a `mut char *` if the caller is responsible for freeing? I'm not too well versed in C APIs but I'm currently forced to do a mut cast to pass into `duckdb_free`